README.md

```diff
@@ -348,6 +349,14 @@ Each indentation level is made up of two spaces. Do not use tabs. (Please set yo
 .c-message {color: #fff; background-color: #000;}
 ```
 
+```
+In the case of single selector, single property rules, one line is acceptable:
+
+/* Fine - one selector, one property */
+.c-alert {font-weight: bold;}
```

Comment:
It's nbd, but should there be spaces inside the curly braces? That's the standard for every other language we use.